azure_pipelines/verify_msalcpp_per_pr_ios.yml

```diff
@@ -53,9 +53,13 @@ jobs:
```

Comment:
This pull request does not update changelog.txt.

Please consider if this change would be noticeable to a partner or user and either update changelog.txt or resolve this conversation.
<!-- changelog-comment -->